Leanplum-SDK/Classes/LeanplumRequest.m

```diff
@@ -729,9 +766,31 @@ + (void)onNoPendingDownloads:(LeanplumVariablesChangedBlock)block
     noPendingDownloadsBlock = block;
 }
 
+/**
+ * Static sendNowQueue with thread protection.
```

Comment:
maybe add more details as to what the sendnowqueue is used for and what its limitations are... same for next method